core/src/main/scala/com/acervera/osm4scala/model/OSMEntity.scala

```diff
@@ -30,10 +30,15 @@ object OSMTypes extends Enumeration {
   val Way, Node, Relation = Value
 }
 
-trait OSMEntity {
+trait OSMEntity extends Product with Serializable {
```

Comment:
extends Product and Serializable??

Reply:
https://nrinaudo.github.io/scala-best-practices/adts/product_with_serializable.html